Article/ArticleService.js

```diff
@@ -0,0 +1,50 @@
+import axios from 'axios'
+
+
+let article_url = 'https://panda-market-api-crud.vercel.app/articles' 
+
+
+export function getArticleList(page_input=1, pageSize_input=10, keyword_input=""){
+    return axios.get(article_url,{params: {
+        page : page_input, 
+        pagesize : pageSize_input,
+        keyword : keyword_input
+    }})
+    .then( (response) => response.data)
+    .catch( (error) => console.error(error));
+}
+
+export function getArticle(articleId){
+    article_url = `https://panda-market-api-crud.vercel.app/articles/${articleId}`
```

Comment:
위에 이미 선언되어있는 변수를 통해 다음과 같이 작성할 수 있어요. baseUrl을 기준으로 뒤의 path만 수정해서 사용해보세요!

```suggestion
    const getArticleUrl = `${article_url}/${articleId}`
```

Reply:
Base url만 바꾸면 더 재사용성이 좋아지고 다양한 환경에서 쓸수 있겠군요! 지적 감사합니다